app/styles/front.scss

```diff
@@ -708,33 +686,60 @@ legend {
   width: 1px;
 }
 
-input:checked + .radioButton, input:checked + .radioButton:active:hover {
+.inputButton {
```

Comment:
Here are (almost) all the radios and checkboxes used in mes-aides. 

Some of them could be changed to buttons for more consistency, but there is still a bunch where a button would not be fitting.